sn_cli/src/files.rs

```diff
@@ -92,30 +32,3 @@ pub fn get_progress_bar(length: u64) -> Result<ProgressBar> {
     progress_bar.enable_steady_tick(Duration::from_millis(100));
     Ok(progress_bar)
 }
-
-fn msg_files_already_uploaded_verified() {
```

Comment:
I've seen these message functions move around so much. I might have to move them into their own rust file.